### DISCUSSION TITLE
window rules

### DISCUSSION THREAD

**Discussion by tomysurya** [labwc/labwc#1904] [Q&A]
i have an issue, when i apply window rules "ResizeTo" for emacs it does resize emacs to the desired dimension,
but now the window position not centered, it goes off to the left top of the screen,
any help will be appreciated. thanks

**Comment by Consolatis** [comment]
You could either use [ResizeRelative](https://labwc.github.io/labwc-actions.5.html#entry_action_name=resizerelative_left=_right=_top=_bottom=) with the same values for all directions or [AutoPlace](https://labwc.github.io/labwc-actions.5.html#entry_action_name=autoplace_policy=value) with `center` after the `ResizeTo` (AFAIR that 2nd one is available only in the master branch and in the upcoming in 0.7.3 release).

**Reply by tomysurya** [reply]
thanks i'll try it

**Comment by 01micko** [comment]
If you know the _size_ of the window (this is for `0.7.2`) you could use something like the following:

```xml
    <windowRule identifier="emacs*" matchOnce="false"> <!-- match app_id or title -->
      <action name="MoveTo" x="685" y="353"/> <!-- depends on screen res -->
      <action name="ResizeTo" width="550" height="375"/> <!-- depends emacs window size -->
    </windowRule>

**Reply by tomysurya** [reply]
this is exactly what i did but it didn't work, but thanks anyway
